FF1Lib/Fun.cs

```diff
@@ -271,20 +271,49 @@ public void ShuffleMusic(MusicShuffle mode, MT19337 rng)
 					//Overwrite beginning of crystal theme with a song that initializes properly but plays no notes
 					Put(0x340C0, Blob.FromHex("FDF805E0D8C7D0C480"));
 
-					List<int> AllSongs = new List<int>
+					List<int> AllSongs = new()
 					{
-						0x7C649, 0x7C6F9, 0x7C75A, 0x7C75B, 0x7C62D, 0x7C75D,

```

Comment:
In my opinion, inflating that list is also not ideal.

Reply:
I agree, this is likely a rule we need to tweak, or we need to add a few exceptions

Reply:
Is it possible to disable the style checker for one line with a comment?